apps/internal/views.py

```
@@ -1,16 +1,28 @@
 import logging
 import traceback
 
-from rest_framework import generics
-from rest_framework import status
+from rest_framework import generics, viewsets, status
```

Ruff (0.11.9)

`rest_framework.viewsets` imported but unused

Remove unused import: `rest_framework.viewsets`

(F401)